src/Snafu.md

```diff
@@ -189,6 +190,19 @@ fn do_something_unique() -> Result<i32, VeryUniqueError> {
 }
 ```
 
+### Delegate to the underlying error
+
+Composing many different error types is idiomatic, can however lead to unwanted nesting, both in the
+string representation and in the source chain. To avoid this, you can use `#[snafu(transparent)]` to
+delegate the `Display` and `Error::source` implementations to the underlying error, effectively
+removing the wrapping error struct or enum variant from the logical error chain.
```

Comment:
First sentence is rough after the comma.

Riffing on ideas...

> Composing different error types is idiomatic, but can lead to unwanted nesting, both in the string representation and in the source chain. 

I think we should circle back here after we have the example as well, which might help solidify more specific cases where this is useful.